src/PropertyList.ts

```diff
@@ -1,91 +1,98 @@
-import { ParticleUtils, SimpleEase, Color } from './ParticleUtils';
+import { combineRGBComponents, SimpleEase, Color } from './ParticleUtils';
 import { PropertyNode } from './PropertyNode';
 
 function intValueSimple(this: PropertyList<number>, lerp: number): number
 {
-    if (this.ease)
-    { lerp = this.ease(lerp); }
+    if (this.ease) lerp = this.ease(lerp);
 
-    return ((this.next.value - this.current.value) * lerp) + this.current.value;
+    return ((this.first.next.value - this.first.value) * lerp) + this.first.value;
 }
 
 function intColorSimple(this: PropertyList<Color>, lerp: number): number
 {
-    if (this.ease)
-    { lerp = this.ease(lerp); }
-    const curVal = this.current.value; const
-        nextVal = this.next.value;
+    if (this.ease) lerp = this.ease(lerp);
+
+    const curVal = this.first.value;
+    const nextVal = this.first.next.value;
     const r = ((nextVal.r - curVal.r) * lerp) + curVal.r;
     const g = ((nextVal.g - curVal.g) * lerp) + curVal.g;
     const b = ((nextVal.b - curVal.b) * lerp) + curVal.b;
 
-    return ParticleUtils.combineRGBComponents(r, g, b);
+    return combineRGBComponents(r, g, b);
 }
 
 function intValueComplex(this: PropertyList<number>, lerp: number): number
 {
-    if (this.ease)
-    { lerp = this.ease(lerp); }
+    if (this.ease) lerp = this.ease(lerp);
+
     // make sure we are on the right segment
-    while (lerp > this.next.time)
+    let current = this.first;
+    let next = current.next;
+
+    while (lerp > next.time)
```

Comment:
This craches if there is only one item on the list. Adding two items with the same value doesn't work since PropertyNode.createList() is too smart and removes an extra item.
I use two items with slightly different values as a workaround.

Reply:
For every behavior that uses a PropertyList, there's a static version for single values - for example, `AlphaBehavior` and `StaticAlphaBehavior`.